homeassistant/components/flume/__init__.py

```diff
@@ -59,14 +78,19 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
     flume_auth, flume_devices, http_session = await hass.async_add_executor_job(
         _setup_entry, hass, entry
     )
+    notification_coordinator = FlumeNotificationDataUpdateCoordinator(
+        hass=hass, auth=flume_auth
+    )
 
     hass.data.setdefault(DOMAIN, {})[entry.entry_id] = {
         FLUME_DEVICES: flume_devices,
         FLUME_AUTH: flume_auth,
         FLUME_HTTP_SESSION: http_session,
+        FLUME_NOTIFICATIONS_COORDINATOR: notification_coordinator,
```

Comment:
I'm confused by the whole coordinator being here?
Why are we using a coordinator and not just fetch the latest always?

Reply:
Flume API has a limit of 120 queries per hour per account. We are already fetching notifications via the coordinator every 5 minutes. In fact I should remove the force parameter. One can use `homeassistant.update_entity` or even `homeassistant.reload_config_entry`.

This is quite similar to the `calendar.list_events` service that returns previously synced data and Google Calendar syncs every 15 minutes.

Reply:
The advantage of keeping the force parameter is it's clearer the service returns cached data and makes it easier to force fetch fresh data.

Reply:
I think the latter is a documentation issue.

Let's remove it. We can always add it back in the future if a use case and bigger need for it raises.

Reply:
Removed